packages/design-system/src/components/web-components/ds-dropdown.test.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { act, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import './ds-dropdown';
+
+const defaultAttrs = {
+  label: 'Dropdown label',
+  options: JSON.stringify([
+    { label: 'Option 1', value: '1' },
+    { label: 'Option 2', value: '2' },
+    { label: 'Option 3', value: '3' },
+  ]),
+};
+
+function renderDropdown(attrs = {}) {
+  return render(<ds-dropdown {...defaultAttrs} {...attrs} />);
+}
+
+describe('Button', () => {
+  it('renders a dropdown', () => {
+    const { asFragment } = renderDropdown();
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('renders disabled dropdown', () => {
+    renderDropdown({ disabled: 'true' });
+    const button = screen.getByRole('button');
+    expect(button).toHaveAttribute('disabled');
+  });
+
+  it('renders disabled', () => {
```

Comment:
what's this test doing that the previous one didn't?

Reply:
nothing to see here!
🧹💨

Reply:
thanks for catching that